features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -99,6 +99,9 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
         state?.config?.ui?.features?.loginAndRegistration?.disabledFeatures);
     const isRuleBasedPasswordExpiryDisabled: boolean = disabledFeatures?.includes("ruleBasedPasswordExpiry");
     const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state?.config?.ui?.features);
+    const maxPasswordValue: number = useSelector((state: AppState) =>
```

Comment:
```suggestion
    const maxPasswordLength: number = useSelector((state: AppState) =>
```

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7185/commits/00a3218230f9b6ccec3de318d8e9ff2645abb7a6 and renamed according to the support PR suggestion from kayathri.